scripts/gen.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+PATH=$PATH:node_modules/.bin
```

Comment:
```suggestion
PATH="${PATH}:node_modules/.bin"
```

can all tokens that contain variables be "quoted" and all variables be ${braced}? just out of pure paranoia, because sh is weird sometimes, and behaves _slightly_ differently on mac.